src/sentry/conf/server.py

```diff
@@ -1451,7 +1451,12 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
 )
 
 # Schedules for taskworker tasks to be spawned on.
-TASKWORKER_SCHEDULES: ScheduleConfigMap = {}
+TASKWORKER_SCHEDULES: ScheduleConfigMap = {
+    "sync_options_trial": {
+        "schedule": timedelta(minutes=5),
+        "task": "sentry.tasks.options.sync_options",
```

Comment:
```suggestion
        "task": "sentry.tasks.options.sync_options",
```
```suggestion
        "task": "options:sentry.tasks.options.sync_options",
```

We'll need the namespace name as well. I forgot about that earlier 🤦